### ISSUE TITLE
Updating epub.js

### ISSUE THREAD

**Issue by aetx** [janeczku/calibre-web#3089]
**tl;dr**: The epub viewer uses epub.js, which is massively outdated. I found a new fork that works, but there are caveats.

**The problem:**
I have the same problem with the navigation of some/most epubs as described in #2221, i.e. the links don't work (and are different than the ones provided in the table of contents contained within the epub itself). As described in that issue, as well as some others referencing similar problems with epub.js (#2078, #2726, #2720, #1284, I am sure there are others), this is an issue with a dependency, i.e. the epub.js library used for rendering epubs. Sadly, this library is out of development for a several years now.

**The "Solution":**
However, I found that a [fork](https://github.com/intity/epub-js) is still getting updates and simply replacing the ``static/js/libs/epub.min.js`` already fixes the table of contents problem. Although, it does introduce other problems, since images no longer load and left/right arrows don't change pages.

I myself am unable to look into this further, because I lack time and knowledge in javascript development. So this is really less of a feature request as a status update and maybe central issue to track issues regarding the outdated epub.js version.

I hope someone from the community can check on this.

P.S.: I just found another epub library called [R2D2BC](https://github.com/d-i-t-a/R2D2BC) that is actively developed and used by Komga.
